charts/immich/templates/microservices.yaml

```diff
@@ -8,7 +8,22 @@ args:
 
 service:
   main:
```

Comment:
nit: let's call the service `metrics`

Reply:
Can it be changed from main? I think main has to be there or else I get an error about the ports not being defined.
```
Release "immich-test" does not exist. Installing it now.
Error: 2 errors occurred:
	* Service "immich-test-microservices" is invalid: spec.ports[0].port: Invalid value: 0: must be between 1 and 65535, inclusive
	* Deployment.apps "immich-test-microservices" is invalid: [spec.template.spec.containers[0].ports[0].containerPort: Required value, spec.template.spec.containers[0].ports[1].name: Duplicate value: "http", spec.template.spec.containers[0].livenessProbe.tcpSocket.port: Invalid value: 0: must be between 1 and 65535, inclusive, spec.template.spec.containers[0].readinessProbe.tcpSocket.port: Invalid value: 0: must be between 1 and 65535, inclusive, spec.template.spec.containers[0].startupProbe.tcpSocket.port: Invalid value: 0: must be between 1 and 65535, inclusive]

```

Reply:
Resolved on discord